gradle-plugins/compose/src/test/kotlin/org/jetbrains/compose/test/tests/integration/GradlePluginTest.kt

```diff
@@ -136,26 +136,49 @@ class GradlePluginTest : GradlePluginTestBase() {
     @Test
     fun nativeCacheKindWarning() {
         Assumptions.assumeTrue(currentOS == OS.MacOS)
-        fun nativeCacheKindWarningProject(kotlinVersion: String) = testProject(
-            TestProjects.nativeCacheKindWarning,
-            defaultTestEnvironment.copy(kotlinVersion = kotlinVersion)
-        )
-
-        val cacheKindWarning = "'kotlin.native.cacheKind' is explicitly set to `none`"
-
-        val args = arrayOf("build", "--dry-run", "-Pkotlin.native.cacheKind=none")
-        with(nativeCacheKindWarningProject(kotlinVersion = TestKotlinVersions.v1_8_20)) {
-            gradle(*args).checks {
-                check.logContains(cacheKindWarning)
+        fun withNativeCacheKindWarningProject(kotlinVersion: String, fn: TestProject.() -> Unit) {
+            with(testProject(
+                TestProjects.nativeCacheKindWarning,
+                defaultTestEnvironment.copy(kotlinVersion = kotlinVersion)
+            )) {
+                fn()
+                testWorkDir.deleteRecursively()
+                testWorkDir.mkdirs()
             }
         }
-        testWorkDir.deleteRecursively()
-        testWorkDir.mkdirs()
-        with(nativeCacheKindWarningProject(kotlinVersion = TestKotlinVersions.v1_9_0) ) {
-            gradle(*args).checks {
-                check.logContains(cacheKindWarning)
+
+        fun testKotlinVersion(kotlinVersion: String) {
+            val args = arrayOf("build", "--dry-run")
+            val commonPartOfWarning = "Compose Multiplatform Gradle plugin manages this property automatically"
+            withNativeCacheKindWarningProject(kotlinVersion = kotlinVersion) {
```

Comment:
May we add a test for the case :"Warnings were also handled incorrectly in this case, because during the configuration of compose-subproject-1 we might set kotlin.native.cacheKind.ios*=none,
which would then cause a warning during the configuration of compose-subproject-2"